app/serializers/export/contact_serializer.rb

```diff
@@ -1,12 +1,28 @@
 module Export
   class ContactSerializer < ActiveModel::Serializer
     attributes :sharing,
-               :receiving,
+               :following,
```

Comment:
Yes, that makes sense 👍 (nevertheless we need to document that somewhere, but that's out of scope for this PR).

(for federation it's clear, because it's only what the sender does, but for export we need to export both sides, so `following` and `followed` makes it clearer :) )